docs/dotnet-five.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What is .NET 5
+description: In this article you will learn about .NET 5, a cross platform, and open-source project that convergences .NET framework and .NET Core together.
+ms.date: 08/31/2020
+ms.topic: overview
+ms.author: dapine
+author: IEvangelist
+---
+
+# What is .NET 5
+
+The .NET ecosystem is being simplified with the convergence of .NET Core and .NET Framework. Naming things is hard, but unifying versions of two disparate .NET implementations can be harder. This article details .NET 5, which can be thought of as .NET Core vNext. With .NET Core concluding at version 3.1 and .NET Framework at version 4.8, to avoid ambiguity the next logical version is .NET version 5.0.
+
+## Evolution of .NET
+
+The advent of .NET Core has evolved the .NET ecosystem in compelling ways. It matured as an open-source project on GitHub, celebrating community contributions, and humbly improving over time. With .NET 5, you get all of the things you loved about .NET Core:
+
+> [!div class="checklist"]
+>
+> - Cross-platform
+> - Open-source and community-oriented
+> - Side-by-side installation
+> - Small project files (SDK-style)
+> - High performance
+> - Command-line tooling
+> - Visual Studio, Visual Studio for Mac, and Visual Studio Code integration
+> - Flexible deployment
+
+In addition to all of these key characteristics, .NET 5 is making significant [performance improvements](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-5).
```

Comment:
I think this can be left out. Perf improvements come with every update, and by the time the majority of people read this doc they'll have already shipped. So it's not worth saying that .NET 5 is making these improvements.